vulnerability/events_creator.go

```diff
@@ -31,7 +31,6 @@ import (
 	libevents "github.com/elastic/beats/v7/libbeat/beat/events"
 	"github.com/elastic/elastic-agent-libs/logp"
 	"github.com/elastic/elastic-agent-libs/mapstr"
-	"golang.org/x/exp/maps"
```

Comment:
Aren't you missing a `"maps"` import?

Reply:
uhhm, yeah, no sure how this passed the pre-commit checks :sweat_smile: